lib/Wikia/src/Service/User/Preferences/PreferenceModule.php

```diff
@@ -27,8 +30,22 @@ public function configure( InjectorBuilder $builder ) {
 				global $wgHiddenPrefs;
 				return $wgHiddenPrefs;
 			} )
-			->bind( PreferenceServiceImpl::DEFAULT_PREFERENCES )->to( function() {
-				return User::getDefaultPreferences();
+			->bind( PreferenceServiceImpl::DEFAULT_PREFERENCES )->to( function( ContainerInterface $c ) {
+				/** @var PreferenceScopeService $scopeService */
+				$scopeService = $c->get( PreferenceScopeService::class );
```

Comment:
Will there be a problem with the lookup of `PreferenceScopeService::class`? Does this create a load order that, if not correct, will cause a failure?